src/main/java/de/neuland/jade4j/util/ArgumentSplitter.java

```diff
@@ -0,0 +1,54 @@
+package de.neuland.jade4j.util;
+
+import java.util.ArrayList;
+import java.util.List;
+
+/**
+ * @author dusan.zatkovsky, 2/5/15
+ */
+public class ArgumentSplitter {
+
+    private static final char argumentDelimiter = ',';
+    private final String arguments;
+    private List<String> argList = new ArrayList<String>();
+    private StringBuffer sb;
+
+    public static List<String> split(String arguments) {
+        return new ArgumentSplitter(arguments).splitArguments();
+    }
+
+    public ArgumentSplitter(String arguments) {
+        this.arguments = arguments;
+    }
+
+    private List<String> splitArguments() {
```

Comment:
This method could be improved in so many ways: 
- StringBuilder is always preffered over a StringBuffer 
- You don't really need to create a private attribute for StringBuffer - it's only appending one element to the arguments list before being discard (you are reinstantiating it every time `pushArg` is called.
- The method itself is confusing. I could help you to refactor it, but I'm afraid to do that without some tests.